xml/islandora_basic_collection_form_mods.xml

```diff
@@ -316,42 +316,101 @@ e.g. Creator, Donor, etc.</description>
         </properties>
         <children/>
       </element>
-      <element name="genre">
+      <element name="genreSet">
         <properties>
-          <type>textfield</type>
+          <type>fieldset</type>
           <access>TRUE</access>
           <collapsed>FALSE</collapsed>
           <collapsible>FALSE</collapsible>
-          <description>A excellent source for genre terms is the Thesaurus for Graphic Materials from the Library of Congress &lt;/br&gt;
-http://www.loc.gov/pictures/collection/tgm/</description>
           <disabled>FALSE</disabled>
           <executes_submit_callback>FALSE</executes_submit_callback>
           <multiple>FALSE</multiple>
           <required>FALSE</required>
           <resizable>FALSE</resizable>
-          <title>Genre</title>
           <tree>TRUE</tree>
-          <actions>
-            <create>
-              <path>self::node()</path>
-              <context>parent</context>
-              <schema/>
-              <type>xml</type>
-              <prefix>NULL</prefix>
-              <value>&lt;genre authority="lctgm"&gt;%value%&lt;/genre&gt;</value>
-            </create>
-            <read>
-              <path>mods:genre</path>
-              <context>parent</context>
-            </read>
-            <update>
-              <path>self::node()</path>
-              <context>self</context>
-            </update>
-            <delete>NULL</delete>
-          </actions>
         </properties>
-        <children/>
+        <children>
+          <element name="genre">
+            <properties>
+              <type>textfield</type>
+              <access>TRUE</access>
+              <collapsed>FALSE</collapsed>
+              <collapsible>FALSE</collapsible>
+              <default_value>lctgm</default_value>
```

Comment:
Looks like this default value might be on the wrong element form field... We're on the _content_ of the `genre` element here... Seems like this default would fit better in the `authority` attribute, below?

Reply:
Should probably be something like: https://github.com/Islandora/islandora_solution_pack_collection/compare/7.x...adam-vessey:7.x-fix-defaults

Reply:
Ack, you're right. Can you make a quick PR to fix this?

Reply:
@bondjimbond: Threw in https://github.com/Islandora/islandora_solution_pack_collection/pull/215